doc/GettingStarted.md

```diff
@@ -222,30 +224,32 @@ launching DAPHNE via Docker (see below) should work the same way as in a native
 
 | tool/lib                             | version known to work (*)    | comment                                                                                                                                 |
 |--------------------------------------|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
-| GCC/G++                              | 9.3.0                        | Last checked version: 12.2                                                                                                              |
 | clang                                | 10.0.0                       |                                                                                                                                         |
 | cmake                                | 3.20                         | On Ubuntu 20.04, install by `sudo snap install cmake --classic` to fulfill the version requirement; `apt` provides only version 3.16.3. |
+| GCC/G++                              | 9.3.0                        | Last checked version: 12.2                                                                                                              |
```

Comment:
I'll change that to 13.2 when merging. I also had it working on gcc 14.1 but that requires dependency updates...